### PR TITLE
v1-update-pr

### PR DESCRIPTION
**Pull request by nbyt3**
Required items, please complete

Change(s):

Initial submission for the cyborgsecurity-hunter-solution
Reason for Change(s):
Updating PR #9041 

See guidance below
Testing Completed:
Yes

Checked that the validations are passing and have addressed any issues that are present:

### CODE REVIEW

**Comment by nbyt3** [comment]
@v-rbajaj Here is the new PR. I closed #9041

**Review by v-rbajaj — APPROVED** [review]
Approved

**Comment by v-rbajaj** [comment]
Hi @nbyt3, we will suggest the required changes by 3rd Nov 2023.

**Comment by v-rbajaj** [comment]
> Hi @nbyt3, we will suggest the required changes by 3rd Nov 2023.

Hi @nbyt3, sorry for the delay, we will provide the changes by 6 Nov 2023.

**Comment by v-rbajaj** [comment]
Hi @nbyt3, please push these changes in create ui and the zip. 
[3.0.0.zip](https://github.com/Azure/Azure-Sentinel/files/13266481/3.0.0.zip)
[createUiDefinition.json](https://github.com/Azure/Azure-Sentinel/files/13266482/createUiDefinition.json)

**Review by v-rbajaj — APPROVED** [review]

**Review by v-atulyadav — DISMISSED** [review]

File: Solutions/Cyborg Security HUNTER/Hunting Queries/Prohibited Applications Spawning cmd.exe or powershell.exe.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+id: fc36d683-385a-4ec2-842d-2982dbed97a4
+name: Prohibited Applications Spawning cmd.exe or powershell.exe
+description: |
+  ''
```
Comment (v-rbajaj) [inline Solutions/Cyborg Security HUNTER/Hunting Queries/Prohibited Applications Spawning cmd.exe or powershell.exe.yaml:4]: Please do add the description for this Hunting Query.

File: Solutions/Cyborg Security HUNTER/ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+| **Version** | **Date Modified (DD-MM-YYYY)** | **Change History**                          |
+|-------------|--------------------------------|---------------------------------------------|
+| 3.0.0       | 03-10-2023                     | Inital Submission
```
Comment (v-rbajaj) [inline Solutions/Cyborg Security HUNTER/ReleaseNotes.md:3]: Please correct typing error for " Inital Submission"

**Comment by nbyt3** [comment]
@v-rbajaj Thank you for the feedback. I just updated the branch.

**Review by v-rbajaj — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**